l10n/configs/pontoon.toml

```diff
@@ -192,6 +192,14 @@ locales = [
         "it",
         "nl",
     ]
+[[paths]]
+    reference = "en/firefox/welcome/page11.ftl"
+    l10n = "{locale}/firefox/welcome/page11.ftl"
+    locales = [
```

Comment:
@peiying2 The full list of target locales is en-US, de, fr, it, nl, es-ES, and en-GB. Of these, I believe de and fr are automatically sent to Smartling so I've only listed it, nl, and es-ES here (same as page10 above). I don't think we need to list en-GB if it can just display the same en file as en-US.

So does this configuration look correct? Do I need to add anything to `vendor.toml` as well, or does it just get ALL Fluent files by default?

Reply:
This looks good!